core/src/test/java/org/elasticsearch/VersionTests.java

```diff
@@ -46,30 +46,30 @@
 public class VersionTests extends ESTestCase {
 
     public void testVersionComparison() throws Exception {
-        assertThat(V_5_3_0.before(V_6_0_0_alpha2), is(true));
+        assertThat(V_5_3_0.before(V_6_0_0_alpha3), is(true));
         assertThat(V_5_3_0.before(V_5_3_0), is(false));
-        assertThat(V_6_0_0_alpha2.before(V_5_3_0), is(false));
+        assertThat(V_6_0_0_alpha3.before(V_5_3_0), is(false));
 
-        assertThat(V_5_3_0.onOrBefore(V_6_0_0_alpha2), is(true));
+        assertThat(V_5_3_0.onOrBefore(V_6_0_0_alpha3), is(true));
         assertThat(V_5_3_0.onOrBefore(V_5_3_0), is(true));
-        assertThat(V_6_0_0_alpha2.onOrBefore(V_5_3_0), is(false));
+        assertThat(V_6_0_0_alpha3.onOrBefore(V_5_3_0), is(false));
 
-        assertThat(V_5_3_0.after(V_6_0_0_alpha2), is(false));
+        assertThat(V_5_3_0.after(V_6_0_0_alpha3), is(false));
         assertThat(V_5_3_0.after(V_5_3_0), is(false));
-        assertThat(V_6_0_0_alpha2.after(V_5_3_0), is(true));
+        assertThat(V_6_0_0_alpha3.after(V_5_3_0), is(true));
 
-        assertThat(V_5_3_0.onOrAfter(V_6_0_0_alpha2), is(false));
+        assertThat(V_5_3_0.onOrAfter(V_6_0_0_alpha3), is(false));
         assertThat(V_5_3_0.onOrAfter(V_5_3_0), is(true));
-        assertThat(V_6_0_0_alpha2.onOrAfter(V_5_3_0), is(true));
+        assertThat(V_6_0_0_alpha3.onOrAfter(V_5_3_0), is(true));
 
         assertTrue(Version.fromString("5.0.0-alpha2").onOrAfter(Version.fromString("5.0.0-alpha1")));
```

Comment:
do we want to add another one of these alpha(N) onOrAfter alpha(N-1) checks? not sure if that is necessary